src/NLog/Targets/ConsoleTarget.cs

```diff
@@ -86,17 +86,23 @@ public sealed class ConsoleTarget : TargetWithLayoutHeaderAndFooter
         [DefaultValue(false)]
         public bool Error { get; set; }
 
-#if !SILVERLIGHT && !__IOS__ && !__ANDROID__
         /// <summary>
         /// The encoding for writing messages to the <see cref="Console"/>.
         ///  </summary>
         /// <remarks>Has side effect</remarks>
         public Encoding Encoding
```

Comment:
I prefer removing it, as it won't do anything. This could be confusing from the API.